routes/api/comments.js

```diff
@@ -0,0 +1,57 @@
+//2 rts
+const express = require('express');
+const {asyncHandler} = require('../utils');
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+const router = express.Router();
+const { CommentLike } = require("../../db/models");
+
+//
+
+//Like a comment
+router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) => {
+
+    const { userId, commentId } = req.body;
```

Comment:
This is unused and can be deleted

Reply:
The contents of line 14 have been deleted.